backend/service/martini/fcc.go

```diff
@@ -58,7 +58,10 @@ func (s *Service) FccContactHandler() rest.Handle {
 func (s *Service) getFrnResponse(ctx context.Context, filerID string) (*v1.FccFrnResponse, error) {
 	if filerID == "123456" {
 		res := new(v1.FccFrnResponse)
-		marshal.DecodeBytes([]byte(testFRN), res)
+		err := marshal.DecodeBytes([]byte(testFRN), res)
+		if err != nil {
```

Comment:
i did not have error validation because we test it a lot and it will be always valid

Reply:
yeah but i feel like does not hurt. For example when removing fields for this PR it proved to be useful (test was failing without knowing why).